drafts/use-cases/4.obtaining-single-event.md

```diff
@@ -0,0 +1,59 @@
+## Obtaining a single event
+
+### Story
+As an application user
+I want to display details of a single event
+So I can find out more about an event I'm interested in.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var event = client.get("/api/events/1");
+// do something with the _event_, i.e. display it in a details view
+```
+
+### Details
+After displaying all the events in a calendar, user should be able to browse details of a selected event.
+It wouldn't be possible for the calendar to show all of the event's attributes,
+thus there is a need to show those outstanding ones in a separate view (i.e. popup, new screen, etc.).
+The application sends a GET request to the selected event's resource identifier:
+
+```
+GET /api/events/1
+
+HTTP 200 OK
+```
+```javascript
+{
+    "@id": "/api/events/1",
+    "eventName": "Event 1",
+    "eventDescription": "Some event 1",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+}
+```
+
+Now the user can see what a given event was about.
+
+### Considerations
+
+#### Expansion
```

Comment:
This would make an impression Hydra is for JSON-LD, while same action can be taken in other formats, for which embedding might not exist.